examples/bank/bank.go

```diff
@@ -64,9 +67,9 @@ func readInt64(call client.Call) (int64, error) {
 	return 0, nil
 }
 
-// moveMoney() moves an amount between two accounts if the amount
-// is available in the from account. Returns true on success.
-func (bank *Bank) moveMoney(from, to []byte, amount int64) bool {
+// moveMoneyTransaction() moves an amount between two accounts in a transaction
```

Comment:
godoc style is to start the comment with the name of the method (as you've done), but with the parentheses.